### PR TITLE
Improved packaging instructions for uploading tests

### PR DESCRIPTION
**Pull request by s-glatshtein**
* Added packaging summary to overview section.
* Added information on how to package the code using Visual Studio publish

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,6 +231,38 @@ Steps are automatically marked as passed or failed, and Scenario Outlines are su
...
+**1. Using Visual Studio:**
+
+* Right-click your solution in the *Solution Explorer* panel and select *Publish...*
+* If this is your first time publishing, create a new publishing target:
+  * Choose *Folder* as your publish target, and press *Next*.
+  * Choose *Folder* again, and press *Next*.
+  * Change the folder your code is built in to `out` and press *Finish*.
+* Press *Publish* next to the *FolderProfile* target to create this folder.
+* Right-click your solution in the *Solution Explorer* panel and select *Open Folder in File Explorer*.
+* Zip the newly created `out` folder.
```
Comment (eldarkra) [inline README.md:252]: Need to better explain the final expected structure of the ZIP file... Should the `out` folder be zipped as well or just it's content?

File: README.md [hunk 2]
```diff
@@ -230,6 +231,38 @@ Steps are automatically marked as passed or failed, and Scenario Outlines are su
...
+
+**2. Using terminal/PowerShell:**
+
+* Open a terminal or powershell window in your solution folder.
+* Run the following command: `dotnet publish -o out <your-solution-file>`
+* Zip the newly created `out` folder.
```
Comment (eldarkra) [inline README.md:258]: Same here

File: README.md [hunk 3]
```diff
@@ -230,6 +231,38 @@ Steps are automatically marked as passed or failed, and Scenario Outlines are su
...
+
+Now your code is ready to be uploaded to TestProject's platform!
+
```
Comment (eldarkra) [inline README.md:261]: Maybe add this link here: https://docs.testproject.io/testproject-sdk/step-by-step-tutorials/c#package-and-upload-tests-to-testproject ?

File: README.md (hunk 1 above)
Comment (s-glatshtein) [reply README.md:252]: Technically you can do either, but it specifically says to zip the folder on purpose.

File: README.md (hunk 2 above)
Comment (s-glatshtein) [reply README.md:258]: Technically you can do either, but it specifically says to zip the folder on purpose.

File: README.md (hunk 3 above)
Comment (s-glatshtein) [reply README.md:261]: I don't follow. It is the same section but with images. Is it relevant to readme?

File: README.md (hunk 1 above)
Comment (eldarkra) [reply README.md:252]: And still, I did not understand if I should include the folder or not...

File: README.md (hunk 2 above)
Comment (eldarkra) [reply README.md:258]: And still, I did not understand if I should include the folder or not...

**Review by eldarkra — APPROVED** [review]

**Pull request merged**